src/main/kotlin/org/neo4j/graphql/BuildingEnv.kt

```diff
@@ -92,20 +92,62 @@ class BuildingEnv(val types: MutableMap<String, GraphQLType>) {
                     else -> addFilterType(getInnerFieldsContainer(typeDefinition), createdTypes)
                 }
 
-                Operators.forType(types[filterType] ?: typeDefinition).forEach { op ->
-                    val wrappedType: GraphQLInputType = when {
-                        op.list -> GraphQLList(GraphQLTypeReference(filterType))
-                        else -> GraphQLTypeReference(filterType)
+                if (field.isRelationship()) {
+                    val list = field.type.isList()
+                    addFilterField(builder, field, RelationOperator.EQ_OR_NOT_EXISTS, filterType,
```

Comment:
can those information bits/methods go into the Operators (or if you want a separate RelationshipOperators) enum: 
- description
- operator
- name

Reply:
since there the descriptions are not static (b/c of `type` and `name`) and there are different descriptions for `n..1` and `n..m`, I would like to keep it in a method.